util/webhook/validator.go

```diff
@@ -136,3 +144,82 @@ func validationResponseFromStatus(allowed bool, status metav1.Status) admission.
 		},
 	}
 }
+
+// EnsureStringSlicesAreEquivalent returns if two string slices have equal lengths,
+// and that they have the exact same items; it does not enforce strict ordering of items.
+func EnsureStringSlicesAreEquivalent(a []string, b []string) bool {
```

Comment:
I think because we don't want to include any functional changes in this PR I'll do this in a follow-up PR (I'll remove the new, still-unused `EnsureStringSlicesAreEquivalent` func from this PR).